core/geometry/src/test/setupTests.ts

```diff
@@ -0,0 +1 @@
+import "../core-geometry"; // Needed to avoid circular import runtime errors with vitest
```

Comment:
please add a comment with a link to the vitest docs you found this in

Reply:
added